Update 01-interfaces.md (#4424)
ianstormtaylor/slate: docs/concepts/01-interfaces.md

}
```

> This is a very permissive interface. All it requires is that the `children` property is defined containing the element's child nodes.

Reviewer: ```suggestion
This is a very permissive interface. All it requires is that the `children` property gets defined containing the element's child nodes.
```

}
```

> The `type` and `url` properties there are your custom API. Slate sees that they exist, but it doesn't ever use them for anything. However, when it goes to render a link element you'll receive an object with the custom properties attached so that you can render it as:

Reviewer: ```suggestion
The `type` and `url` properties are your custom API. Slate sees that they exist, but doesn't use them. However, when Slate renders a link element, you'll receive an object with the custom properties attached so that you can render it as:
```

```jsx
<a href={element.url}>{element.children}</a>
```

> When getting started with Slate, it's important to understand all of the interfaces it defines. There is a handful of them that are discussed in each of the guides.

Reviewer: ```suggestion
When getting started with Slate, it's important to understand all of the interfaces it defines. There are a handful of interfaces that are discussed in each of the guides.
```

I think are is more accurate... easier to see when changing the word them to interfaces

}
```

> There are lots of helper functions available for all of the common use cases when working with different interfaces. When getting started it pays to read through them all because you can often simplify complex logic into just a handful of lines of code with them.

Reviewer: ```suggestion
There are many helper functions available for all common use cases when working with different interfaces. When getting started it helps to read through all of them so you can often simplify complex logic into just a handful of lines of code.
```
